projects/plugins/jetpack/extensions/shared/components/inspector-notice/style.scss

```
@@ -0,0 +1,13 @@
+.jetpack-inspector-notice {
+    padding: 24px;
+    margin: 0 16px 24px;
+    background: #f2f2f2;
```

Maybe we should use a core color here? `$gray-100` maybe?

```suggestion
@import '../styles/gutenberg-base-styles.scss';

.jetpack-inspector-notice {
	padding: 24px;
	margin: 0 16px 24px;
	background: $gray-100;
```

---

Thank you. `$gray-100` is `#f0f0f0`. Pretty close to `#f2f2f2`. Done. https://github.com/Automattic/jetpack/pull/22694/commits/c275adf5104dce9b1d8291cc25a3066d743be6cc